<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01HW247159" name="com.TestSuite.Payments.MyDesk_PayMob_001" tests="1" failures="1" timestamp="21 Dec 2016 08:25:25 GMT" time="91.017" errors="0">
  <testcase name="MyDesk_PayMob_001_Method" time="91.017" classname="com.TestSuite.Payments.MyDesk_PayMob_001">
    <failure type="java.lang.AssertionError" message="null script failed, Case Category - Payment - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Case Category - Payment - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClickNotAssertive(WebActions.java:899)
	at com.BusinessModules.MyDesk.Cases_Page.select_Action_Category_for_Cases(Cases_Page.java:339)
	at com.TestSuite.Payments.MyDesk_PayMob_001.MyDesk_PayMob_001_Method(MyDesk_PayMob_001.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_PayMob_001_Method -->
</testsuite> <!-- com.TestSuite.Payments.MyDesk_PayMob_001 -->
